nupic/algorithms/anomaly.py

```diff
@@ -123,25 +126,9 @@ def compute(self, activeColumns, predictedColumns,
                               	(used in anomaly-likelihood)
     @return the computed anomaly score; float 0..1
```

Comment:
From @breznak's comment below regarding broken compatibility:
> Well, it would change results, but I'm not sure how crucial (=how many ppl are using it) it is. Anyway, the break would be easy to fix. Motivation is that you can use the Anomaly class in Likelihood mode for the "original" anomaly_likelihood code which is used in more cases.

What might be returned here is no longer compatible with the documentation `@return the computed anomaly score; float 0..1`. In some cases, this will be a score (e.g., raw anomaly score), in other cases it will be the probability (e.g., from the new `computeLikelihood`), and something else entirely when the user passes in a custom function for computing anomaly.